httpservice/CMakeLists.txt

```diff
@@ -25,12 +25,12 @@ else()
 endif()
 
 set(_private_headers
-  src/HttpOutputStreamBuffer_p.h
-  src/HttpServlet_p.h
-  src/HttpServletRequest_p.h
-  src/HttpServletResponse_p.h
-  src/ServletConfig_p.h
-  src/ServletContainer_p.h
+  src/HttpOutputStreamBuffer.h
+  src/HttpServlet.h
```

Comment:
An extension of my comment about header guards: there are also five headers from httpservice that have the same file name, one public and one private.